mysql-test/suite/innodb/t/instant_alter_crash.test

```diff
@@ -225,6 +225,33 @@ INSERT IGNORE INTO t1 (f3) VALUES ( 'b' );
 INSERT IGNORE INTO t1 (f3) VALUES ( 'l' );
 SET DEBUG_SYNC="now SIGNAL con1_finish";
 
+connection default;
+reap;
+CHECK TABLE t1;
+DROP TABLE t1;
+
+--echo #
+--echo # MDEV-19044 Alter table corrupts while applying the
+--echo #             modification log
+--echo #
```

Comment:
I think that this test as well as the preceding test for MDEV-26198 would better belong to the test file `innodb.instant_alter_debug`, which is for debug instrumented tests that do not require the server to be restarted. Unlike this test, that test could also run with `./mtr --embedded`.

The purpose of this test file is to cover crash recovery. However, I see that already from its initial revision a4948dafcd7eee65f16d848bdc6562fc49ef8916 this test did require a debug instrumented build. So, the only thing that we are losing here is some `./mtr --embedded` coverage on debug instrumented builds.